### ISSUE TITLE
Add callback function after scroll is ended

### ISSUE THREAD

**Issue by romanlex** [flyingant/react-scroll-to-component#6]
Example focus on input after scrolling

**Comment by chanthafef** [comment]
maybe specifying a duration and using a setTimeout function is good enough

**Comment by sag1v** [comment]
A callback has a lot of use cases indeed.
I would say any animation library should expose a life cycle methods eg: `before-animation` `after-animation` etc.. but a callback for a done animation is a great start.

**Comment by sag1v** [comment]
@flyingant 
I've noticed that `react-scroll-to-component` uses `scroll-to` behind the scenes.  
`scroll-to` uses `tween.js`, and i came across [this comment](https://github.com/component/scroll-to/issues/1#issuecomment-358056653).
So i guess this is "doable" :)

**Comment by flyingant** [comment]
@sag1v 
Yes, of course. it's native supported. I will put some examples to the demo.
